src/Pages/Auth/Login/index.js

```diff
@@ -1,11 +1,35 @@
-import React from "react"
+import React, { useEffect } from "react"
+import { Link } from "react-router-dom"
 
 Login.propTypes = {}
 
-function Login() {
+function Login(props) {
+  const loggedStatous = props.userIsLogged
+
+  function login(loggedStatous) {
+    props.setUserIsLogged(true)
+    window.location.reload(false)
+  }
+
+  function logout(loggedStatous) {
+    props.setUserIsLogged(false)
+    window.location.reload(false)
```

Comment:
See this post https://dev.to/projectescape/programmatic-navigation-in-react-3p1l
and use the useHistory hook in order to navigate to the login page

Reply:
should it be better if the app navigates to the homepage after login or logout? 
(instead of the user clicking the 'go back' button )

Reply:
do you mean i make it in a way that when user clicks login or logout he will be automatically returned to home page instead of having to use a button?